docs/guide/learning/how-to-learn-vue.md

```diff
@@ -243,6 +243,256 @@ In this section, we will go over the most popular technologies and the struggles
 
 ### Laravel
 
+#### Reasons to use Laravel and Vue
+
+##### Scaffolded by default
+
+Laravel [by default comes with everything you need](https://laravel.com/docs/5.7/frontend#writing-vue-components) to get up and running with Vue.
```

Comment:
I would skip the version, so its always the newest.

Reply:
Makes sense, consider it done.